src/pages/EditBooks/EditBooks.tsx

```diff
@@ -60,6 +61,9 @@ export const EditBooks = () => {
 
     return (
         <>
+        <div className="hamburger-menu-container">
```

Comment:
if you created HamburgerMenu you can move this div into this component

Reply:
Done